packages/core/src/fhir-deduplication/deduplicate-fhir.ts

```diff
@@ -248,20 +244,42 @@ const medicationRelatedFiltersMap = new Map<string, ResourceFilter>([
 
 const encounterFiltersMap = new Map<string, ResourceFilter>([
   ["Condition", removeDanglingReferences],
+  ["Location", removeDanglingReferences],
+  ["Practitioner", removeDanglingReferences],
+]);
+
+const conditionsFiltersMap = new Map<string, ResourceFilter>([
+  ["Practitioner", removeDanglingReferences],
 ]);
 
 const diagReportFiltersMap = new Map<string, ResourceFilter>([
   ["Observation", removeDanglingReferences],
   ["Encounter", removeDanglingReferences],
+  ["Practitioner", removeDanglingReferences],
+  ["Organization", removeDanglingReferences],
+]);
+
+const observationFiltersMap = new Map<string, ResourceFilter>([
+  ["Practitioner", removeDanglingReferences],
+  ["Organization", removeDanglingReferences],
+]);
+
+const procedureFiltersMap = new Map<string, ResourceFilter>([
+  ["Practitioner", removeDanglingReferences],
+  // ["Organization", removeDanglingReferences],
```

Comment:
why commented out? I see both Orgs and Practitioners in Procedures.hbs

Reply:
Accident, i think 😓